ui/DesignSystem/Component/List.svelte

```diff
@@ -1,11 +1,9 @@
 <script>
```

Comment:
Is it really a general list now? It still uses references like `project-card`?

Reply:
Forgot to remove that unused class... But yeah, apart from that it's pretty generic.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/361/commits/1918f137453a68839171a7ac8dcd1ea6f40afa37